content/games/factions/routines.md

```diff
@@ -0,0 +1,143 @@
+     the Defensive trait, they must be evenly dispersed amongst Group clusters.
+
+### Ambitions
+
+Routine Companies may have [Ambitions]({{% ref "/games/factions/rules#ambitions" %}}). For each
+Ambition listed for the Personality, roll a d6: on a 4+, they have that Ambition. Routine Company
+Ambitions always count as revealed.
+
+- **Arcane:** Give Better Than We Get, Draw First Blood,They Will Know My Name
+- **Bold:** They Will Reel Before Us, Give Better Than We Get, Draw First Blood, They Will KNow My
+  Name
+- **Cautious:** We Will Not Break, Give Better Than We Get, Rain Hell Upon Them
+- **Ferocious:** They Will Reel Before Us, We Will Snap Their Spine, Break Even Their Anvil, Draw
+  First Blood, We Will All Shed Blood
+- **Guarded:** We Will Not Break, Win Without Moving, Do Not Waver, Give Better Than We Get, Rain
+  Hell Upon Them
+- **Incompetent:** None.
+
+## Turns
+
+Routine Companies start their turn as normal, rallying Shaken Groups then testing for Reckless
+Groups.
+
+When activating individual Groups, unless specified, they follow this process:
+
+1. Must activate the Captain's Group first, then the closest Group to the Captain's Group which has
+   not yet activated.
+1. Roll 1d6 to determine the activation: Move (1--2), Attack (3--4), Shoot (5--6). If the group
+   cannot perform the specified action, default to the next lower-order action until they can.
+   - If Moving, roll 1d6 and: Move away from nearest enemy (1--2), Move towards nearest enemy
+     (3--4), Move towards nearest objective (5--6).
+1. Resolve each activation as normal.
+
+### Arcane Group Activations
+
+Arcane Companies must activate any Groups with the Casting or Summoner order before others, from
+highest to lowest point value. Groups capable of the Casting or Summoning order _must_ use that
+order. If capable of both, alternate between them each time.
+
+### Bold Group Activations
+
+Bold Companies must activate the Group nearest an enemy or objective that has not already activated.
+Groups capable of seizing an objective or attacking an enemy must do so, preferring objectives over
+attacking.
+
+### Cautious Group Activations
+
+Cautious Companies may not Attack an enemy Group whose R is equal to or greater than their own.
+Groups more than 6" away from a friendly Group must Move towards one if possible.
+
+### Ferocious Group Activations
+
+Ferocious Companies must activate the Group nearest an enemy that has not already activated. Groups
+capable of Attacking or Shooting must do so, preferring whichever activation order has a better hit
+chance. Groups not within range of an objective or enemy must Move towards the nearest enemy Group.
+
+### Guarded Group Activations
+
+Guarded Companies activate as normal but may not activate to Attack an enemy Group unless a friendly
+Group is within 12". Groups not within 12" of a friendly Group must move towards the nearest
+friendly Group or defensible position.
```

Comment:
```suggestion
Guarded Companies may not activate to Attack an enemy Group unless there is a friendly
Group within 12". Groups not within 12" of a friendly Group must move toward the nearest
friendly Group if possible, or toward the nearest defensible position otherwise.
```